### ISSUE TITLE
HeaderCheck doesn't work correctly

### ISSUE THREAD

**Issue by Shryne** [teamed/qulice#936]
I get this error from CheckStyle: 
> [INFO] Checkstyle: [...]\goop\src\main\java\testing\MyTest.java[4]: Line does not match expected header line of ' *'. (HeaderCheck)
  
Even though line 4 does start with a space and a *. Here are the files:  
[MyTest.java](https://github.com/Shryne/goop/blob/config/src/main/java/testing/MyTest.java)  
[LICENSE.txt](https://github.com/Shryne/goop/blob/config/LICENSE.txt)  
[pom.xml](https://github.com/Shryne/goop/blob/config/pom.xml)  

Additionally, I get this error for my package-info.java files:
> [INFO] Checkstyle: [...]\goop\src\main\java\testing\package-info.java[1]: Missing a header - not enough lines in file. (HeaderCheck)

I guess HeaderCheck wants to say that I don't have a license comment there, but that's not the case:  [package-info.java](https://github.com/Shryne/goop/blob/config/src/main/java/testing/package-info.java)

Maybe checkstyle doesn't like the different line endings. All files have the unix line ending (/n) except for the LICENSE.txt which has the windows line ending (\r\n). I tryed to use the unix line ending for the license file too, but in that case I get this error:
> [INFO] ------------------------------------------------------------------------
[INFO] Building goop 1.1.0
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- qulice-maven-plugin:0.18.1:check (default-cli) @ goop ---
[INFO] Starting FindBugs validator
[INFO] >> WARNING: An illegal reflective access operation has occurred
[INFO] >> WARNING: Illegal reflective access by org.dom4j.io.SAXContentHandler (file:/C:/Users/Steven/.m2/repository/dom4j/dom4j/1.6.1/dom4j-1.6.1.jar) to method com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser$LocatorProxy.getEncoding()
[INFO] >> WARNING: Please consider reporting this to the maintainers of org.dom4j.io.SAXContentHandler
[INFO] >> WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[INFO] >> WARNING: All illegal access operations will be denied in a future release
[INFO] >> No classfiles specified; output will have no warnings
[INFO] >> The following errors occurred during analysis:
[INFO] >>   File not found: filesystem:C:\DATA\java%20stuff\goop\target\classes
[INFO] >>   Error scanning testing/MyTest for referenced classes
[INFO] >>     java.lang.IllegalArgumentException
[INFO] >>       At org.objectweb.asm.ClassReader.<init>(Unknown Source)
[INFO] >>       At org.objectweb.asm.ClassReader.<init>(Unknown Source)
[INFO] >>       At edu.umd.cs.findbugs.asm.FBClassReader.<init>(FBClassReader.java:35)
[INFO] >>       At edu.umd.cs.findbugs.classfile.engine.asm.ClassReaderAnalysisEngine.analyze(ClassReaderAnalysisEngine.java:48)
[INFO] >>       At edu.umd.cs.findbugs.classfile.engine.asm.ClassReaderAnalysisEngine.analyze(ClassReaderAnalysisEngine.java:34)
[INFO] >>       At edu.umd.cs.findbugs.classfile.impl.AnalysisCache.getClassAnalysis(AnalysisCache.java:262)
[INFO] >>       At edu.umd.cs.findbugs.classfile.engine.ClassInfoAnalysisEngine.analyze(ClassInfoAnalysisEngine.java:75)
[INFO] >>       At edu.umd.cs.findbugs.classfile.engine.ClassInfoAnalysisEngine.analyze(ClassInfoAnalysisEngine.java:38)
[INFO] >>       At edu.umd.cs.findbugs.classfile.impl.AnalysisCache.getClassAnalysis(AnalysisCache.java:262)
[INFO] >>       At edu.umd.cs.findbugs.FindBugs2.buildReferencedClassSet(FindBugs2.java:774)
[INFO] >>       At edu.umd.cs.findbugs.FindBugs2.execute(FindBugs2.java:222)
[INFO] >>       At com.qulice.findbugs.Wrap.run(Wrap.java:132)
[INFO] >>       At com.qulice.findbugs.Wrap.main(Wrap.java:70)
[INFO] >> The following classes needed for analysis were missing:
[INFO] >>   java.lang.Object
[INFO] >>   javax.accessibility.Accessible
[INFO] >> 
[INFO] Finishing FindBugs validator
[INFO] Calling org.apache.maven.plugins:maven-enforcer-plugin:1.0-beta-1:enforce...
[INFO] Calling com.ning.maven.plugins:maven-duplicate-finder-plugin:1.0.7:check...
[INFO] Checking compile classpath
[INFO] Checking runtime classpath
[INFO] This is not an SVN project
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 7.912 s
[INFO] Finished at: 2018-10-23T15:15:44+02:00
[INFO] Final Memory: 19M/70M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.qulice:qulice-maven-plugin:0.18.1:check (default-cli) on project goop: Execution default-cli of goal com.qulice:qulice-maven-plugin:0.18.1:check failed. IllegalArgumentException -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.

So in this case, it doesn't even build.

**Comment by Shryne** [comment]
I was able to get rid of the error. This is what I did:  
1. I changed the line ending of the license file to \n. As a result, I got the message from above. The message comes from findbugs. I guess findbugs works after pmd and checkstyle and only if they don't find anything.  
2. I used mvn:clean. Findbugs still complains about:
> [INFO] >> The following classes needed for analysis were missing:
[INFO] >>   java.lang.Object
[INFO] >>   javax.accessibility.Accessible

But the BUILD is successful, so I guess it's fine now.
